app/models/orm/migrations/migrate_asset_tables.py

```diff
@@ -0,0 +1,91 @@
+"""Add version column to all data tables."""
```

Comment:
All this migration stuff is just for revisions?

Reply:
Yup.